### ISSUE TITLE
Let's get gridsome SPONSORED?

### ISSUE THREAD

**Issue by dubem-design** [gridsome/gridsome#1632]
## Description

Gridsome is an incredible tool, but it has not enjoyed any updates for a long time now. I know many companies and startups have benefited from it. We can get it sponsored and get some guys working on it full-time if we work together. 

Please drop a comment if you agree, let's amplify. 

Humble Regards,
Dubem

This is the ultimate bug!!! 😭

**Comment by kleinerriese** [comment]
Yes! Would love to stay with Gridsome, however if there is no progress we might need to change to Nuxt...

**Comment by OzzyTheGiant** [comment]
Yeah I'm very concerned at this point. I mean I understand we all just went through the pandemic, personal problems, Vue 3 breaking the ecosystem, and even some developers are affected by the war, but I want reassurance that this project can make it to 1.0 and will be available for a long time.

Otherwise, I'm currently eyeing Astro (supports Vue and Svelte integrations, which I prefer over that wretched React/Gatsby), or Hugo (happened to pick up Go a while back). Then there's SvelteKit but it's not even out of beta yet and not sure how SSG performs over there.

**Comment by fbuncom4** [comment]
> Yes! Would love to stay with Gridsome, however if there is no progress we might need to change to Nuxt...

Contemplating the same move 😢

**Comment by FilledStacks** [comment]
Anyone still holding out?

My experience is not in webdev so I'm not very eager to move over. Have to make sure urls match, styling of new site etc. 

Did anyone migrate their website? What did you use and what was the effort involved?

**Comment by fbuncom4** [comment]
@FilledStacks I'm still holding out on one website. I have built a new website using Gatsby (React-based) which was fine. I haven't switched my last website to Nuxt yet because it was more work than I wanted to do. But Nuxt3 recently pushed a stable release with more documentation so it could be easier now!

**Comment by FilledStacks** [comment]
@fbuncom4 Did you have any issues keeping your urls exactly the same as your current structure? I get about 6-10k visitors a month, I don't want to lose valuable links to my content and my indexing score on google. 

Uuuugggghhhh ... Open source is hard (as a maintainer myself). I wish there was a proper open source business course that we can all take.

**Comment by fbuncom4** [comment]
@FilledStacks no, I didn't have issues on that front but had to be mindful.

I wish you the best with your journey 🙏🏾

**Comment by mathieubellon** [comment]
> Yes! Would love to stay with Gridsome, however if there is no progress we might need to change to Nuxt...

I am coming from Nuxt and nuxt/content, 6mn to build a 221 pages documentation is quite a problem in my opinion. For a blog / small website or app it is perfect - I eyed to gridsome but this thread makes me think I need to continue my quest.

**Comment by dubem-design** [comment]
I have moved on. I use Astrojs now. You can achieve 3mins build time for over 400 pages.

**Comment by chrisrueger** [comment]
> I use Astrojs now.

Can you elaborate a bit on your migration? 
What was the biggest effort / pain? 

How difficult was it to replace the GraphQL layer from Gridsome with stuff from AstroJS?

**Comment by kleinerriese** [comment]
We have indeed moved on to Nuxt 3 in combination with Strapi. Builds over 300 pages in under a minute (SSR enabled). 
I am quite happy. 
For documentation you might want to check out Vuepress? It is exactly designed for that usecase...
